requirements-hw.txt

```diff
@@ -0,0 +1,6 @@
+Adafruit-BNO055==1.0.2
+Adafruit-GPIO==1.0.1
+Adafruit-PCA9685==1.0.1
+pyserial==3.2.1
+RPi.GPIO==0.6.3
+smbus-cffi==0.5.1
```

Comment:
So this is added for hw/pi specific things. Unless you're on a pi, this would probably fail to install.